app/assets/javascripts/controllers/configuration_manager_form_controller.js

```diff
@@ -0,0 +1,113 @@
+ManageIQ.angular.app.controller('configurationManagerFormController', ['$http', '$scope', 'configurationManagerFormId', 'miqService', 'configurationManagerService', 'url', function($http, $scope, configurationManagerFormId, miqService, configurationManagerService, url) {
+  var vm = this;
+
+  vm.configurationManagerModel = angular.copy(configurationManagerService.managerModel);
+
+  vm.postValidationModel = {};
+
+  vm.formId = configurationManagerFormId;
+  vm.afterGet = false;
+  vm.saveable = miqService.saveable;
+  vm.validateClicked = configurationManagerService.validateClicked;
+  vm.modelCopy = angular.copy(vm.configurationManagerModel);
+  vm.model = 'configurationManagerModel';
+  vm.checkAuthentication = true;
+
+  ManageIQ.angular.scope = vm;
+
+  miqService.sparkleOn();
+  if (configurationManagerFormId === 'new') {
+    vm.newRecord = true;
+
+    vm.configurationManagerModel.name = '';
+    vm.configurationManagerModel.url = '';
+    vm.configurationManagerModel.verify_ssl = false;
+    vm.configurationManagerModel.default_userid = '';
+    vm.configurationManagerModel.default_password = '';
+
+    $http.get(url + '/form_fields/' + configurationManagerFormId)
+      .then(getAutomationManagerNewFormDataComplete)
```

Comment:
maybe rename these *automationManager*  to use the `configurationManager` as well?

Or .. even better.. just drop the prefix, there is absolutely no point in calling something *configurationManager* when *in* configuration manager controller :). (Perhaps except for the model name itself, and for the injected bits.)

(Also, the get- prefix doesn't make sense for a method which does no getting and is not a getter.)

So.. I propose `newFormDataComplete`, `editFormDataComplete` and `editButtonClicked`.